src/ewatercycle/forcing/_lisvap.py

```diff
@@ -0,0 +1,154 @@
+"""Generate lisvap files to be used for lisflood.
+
+example:
+from .lisvap import create_lisvap_config, run_lisvap
+config_file = create_lisvap_config(parameterset, start_time, end_time, directory, forcing_files)
+run_lisvap(version, config_file, parameterset, directory)
+"""
+
+import os
+import subprocess
+from pathlib import Path
+from typing import Tuple
+
+from ewatercycle import CFG
+from ..models.lisflood import XmlConfig
```

Comment:
This is causing circular import. Can you move XmlConfig to own python module to break circle